ui/src/app/job-details/job-details.component.ts

```diff
@@ -26,8 +26,8 @@ export class JobDetailsComponent implements OnInit {
   }
 
   hasTabs(): boolean {
-    if (this.job.inputs || this.job.outputs) {
-      return true;
+    if (( this.job.inputs && Object.keys(this.job.inputs).length !== 0) || (this.job.outputs  && Object.keys(this.job.outputs).length !== 0)) {
```

Comment:
Is this change related?

Reply:
tangentially – this function will determine if the tab group, which the timing diagram is in, will be displayed